base-theme/layouts/partials/head.html

```diff
@@ -1,4 +1,6 @@
 {{- $gtmId := getenv "GTM_ACCOUNT_ID" -}}
+{{- $metaDataDescription := "MIT OpenCourseWare is a web-based publication of virtually all MIT course content. OCW is open and available to the world and is a permanent MIT activity" -}}
+{{- $metaDataKeywords := "MIT, OpenCourseWare, OCW, MIT activity, Online Education" -}}
```

Comment:
@pdpinch I also added some keywords, can you proofread them and provide some more?

Reply:
We can reuse the existing ones on ocw.mit.edu (if there aren't too many) 

```suggestion
{{- $metaDataKeywords := "[opencourseware,MIT OCW,courseware,MIT opencourseware,Free Courses,class notes,class syllabus,class materials,tutorials,online courses,MIT courses]()" -}}
```